packages/ddp-server/crossbar.js

```diff
@@ -87,31 +96,125 @@ _.extend(DDPServer._Crossbar.prototype, {
     var self = this;
 
     var collection = self._collectionForMessage(notification);
+    var listenersForCollection = self.listenersByCollection[collection];
+    var bufferForCollection = self.buffersPerCollection[collection];
 
-    if (! _.has(self.listenersByCollection, collection)) {
-      return;
+     if (listenersForCollection) {
```

Comment:
I'm not sure whether we even still need the listeners (we could also refactor any other usages of listener to make use of buffer instead, I think).